apps/coscrad-frontend-e2e/src/e2e/resources/audio-items/audio-annotation-flow.e2e.cy.ts

```diff
@@ -0,0 +1,218 @@
+import { AggregateType, MIMEType, ResourceType } from '@coscrad/api-interfaces';
+import { buildDummyAggregateCompositeIdentifier } from '../../../support/utilities';
+
+const audioTitleInLanguage = 'Audio Title (Lang)';
+
+const basicAudioAggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.audioItem,
+    1
+);
+
+const mediaItemCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    21
+);
+
+const { id: basicAudioId } = basicAudioAggregateCompositeIdentifier;
+
+const audioDetailRoute = `/Resources/AudioItems/${basicAudioId}`;
+
+const validUrl =
+    'https://coscrad.org/wp-content/uploads/2023/05/metal-mondays-mock2_370934__karolist__guitar-solo.mp3';
+
+const buildDetailRoute = (audioItemId: string) => `/Resources/AudioItems/${audioItemId}`;
+
+const millisecondsToRoundedSeconds = (milliseconds: number) => {
```

Comment:
For this helper and `isWithinToleranceOfFloat`, if they were in a separate lib with their own tests, could we call them in this cypress test?  or is it bad form to use external helpers?

Reply:
See [the docs](https://docs.cypress.io/guides/references/assertions). `should('be.closeTo')`

Reply:
I see that you've done this. Great work!